app/views/layouts/_head.haml

```diff
@@ -12,7 +12,7 @@
   = render :partial => 'analytics/head_scripts', locals: { community: @current_community, user: @current_user, plan: @current_plan }
 
   :css
-    @import url('https://fonts.googleapis.com/css?family=Montserrat+Alternates:400,600,700&display=swap');
+    @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;700&display=swap');
```

Comment:
I checked the stylesheets and they use font-weight values from 300 until 800. We need to download these too or the design will break.